target.go

```diff
@@ -173,11 +204,19 @@ func (s *targetServer) createObliviousResponseForQuery(query *odoh.ObliviousDNSQ
 }
 
 func (s *targetServer) obliviousQueryHandler(w http.ResponseWriter, r *http.Request) {
+	requestReceivedTime := time.Now()
```

Comment:
Can we put all metrics collection code behind a flag? Or at least put the elastic search ingestion behind a flag?